### ISSUE TITLE
There is no category "Reaction" in the current data model

### ISSUE THREAD

**Issue by jmrohwer** [StephanM87/Strenda-biocatalysis#46]
The current data model contains no category "Reaction". In fact the data model describes in great detail the composition and setup of the reactor and reaction conditions, including all components, but the reaction information itself is missing. I think this is important metadata to be included, for the following reasons:

1. Every biocatalyst catalyses a reaction (or more than one), and the purpose of the reactor is to make this reaction happen. It should therefore be specified.
2. Every reaction has reactants and products. These should be listed separately (see ) and linked to the reaction. In this way functional relationships are captured in the metadata.
3. This becomes even more important when wanting to report kinetic information, as this pertains to the rate of change of reactants and products.

**Comment by PeterHalling** [comment]
Yes, I agree, this is a valuable category, that sits a bit between the metadata of experiment set-up and the results that we are going to provide for in Measurement. The experimenter will almost always have an idea in advance of the expected/desired reaction. That will lead to decisions about starting materials to be included in the Components. The predicted reaction will also lead to decisions about what to measure (and thus report in Measurement). These measurements may lead to revised views about what reaction(s) are happening. That highlights one essential feature of a Reaction category - it should allow for more than one significant reaction to be reported.

**Comment by Peacock68** [comment]
Well, on the one hand, I'd agree with you both. But on the other hand, actually, a reaction is already defined in the intial STRENDA Guidelines. Given that the Biocatalysis Guidelines are an extension of the "Root" Guidelines, we do not need a reaction definition if there isn't more to specify. I believe that we must be clear ourselves which role the Biocatalysis Guidelines play: Is it an extension or is it a stand-alone Guidelines? I'd be fine with both (with slight preferences to the latter one with respect to the usability of the guidelines).
